### PR TITLE
Routers, Implement CRUD endpoints for Seekers (Employees) model.

### PR DESCRIPTION
**Pull request by jesuarva**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: backEnd/server/routes.js [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 
 module.exports = function(server) {
   /*
     ADD API ROUTES TO SERVER
 
     Ex:
     server.use('/api/tasks', tasksRouter);
-    server.use('/api/users', usersRouter);
   */
+  server.use('/api/employees', seekersRouter);
```
Comment (d-vail) [inline backEnd/server/routes.js]: The endpoint should be /api/seekers

Comment (jesuarva) [reply backEnd/server/routes.js]: annotated!

File: backEnd/api/routes/Seeker.router.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+
+const Users = require('../models/Seeker.model');
```
[the remark this answers is not in the document]
Comment (jesuarva) [reply backEnd/api/routes/Seeker.router.js]: done!

File: backEnd/api/routes/Seeker.router.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const { RouterFactory } = require('express-router-factory');
+
+const router = express.Router();
+
+const RF = new RouterFactory(router, Users);
```
[the remark this answers is not in the document]
Comment (jesuarva) [reply backEnd/api/routes/Seeker.router.js]: All suggestions are done.

**Pull request merged**